src/test/java/org/jnode/fs/xfs/XfsFileSystemTest.java

```diff
@@ -139,7 +139,7 @@ public static void buildXfsMetaDataStructure(FSEntry entry, StringBuilder actual
      * @param indent the indent level.
      *
      */
-    private static StringBuilder getXfsMetadata(FSEntry entry, StringBuilder actual,String indent) throws IOException {
+    private static StringBuilder getXfsMetadata(XfsEntry entry, StringBuilder actual,String indent) throws IOException {
         actual.append(indent);
         actual.append(indent);
         actual.append("atime : " +  getDate(((FSEntryLastAccessed) entry).getLastAccessed() ));
```

Comment:
You should no longer need to cast entry on this line and the next, as `XfsEntry` implement these interfaces.

Reply:
HI , Erwin your suggestion was applied . Thanks